cache/lru_cache.cc

```diff
@@ -375,6 +378,43 @@ Status LRUCacheShard::InsertItem(LRUHandle* e, Cache::Handle** handle) {
   return s;
 }
 
+void LRUCacheShard::Promote(LRUHandle* e) {
```

Comment:
So, after promotion, we do not need to deal with the data in the secondary cache? For example, erase it, or  lower its priority?